x-pack/plugins/endpoint/server/routes/resolver/children.ts

```diff
@@ -4,18 +4,18 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import _ from 'lodash';
 import { schema } from '@kbn/config-schema';
 import { RequestHandler, Logger } from 'kibana/server';
-import { extractEntityID } from './utils/normalize';
-import { getPaginationParams } from './utils/pagination';
-import { LifecycleQuery } from './queries/lifecycle';
-import { ChildrenQuery } from './queries/children';
+import { Fetcher } from './utils/fetch';
 import { IndexPatternRetriever } from '../../index_pattern';
 
 interface ChildrenQueryParams {
-  after?: string;
-  limit: number;
+  // the rough approximation of how many children you can request per process, per level
```

Comment:
❔ It would be nice to have this "approximation" governed by some kind of performance measurement.